static/app/views/performance/traces/content.tsx

```diff
@@ -119,6 +128,13 @@ export function Content() {
         <EnvironmentPageFilter />
         <DatePageFilter defaultPeriod="2h" />
       </PageFilterBar>
+      {hasMetric && (
+        <StyledAlert type="info" showIcon>
+          {tct('The metric query [metricQuery] is filtering the results below.', {
+            metricQuery: <strong>{`${metricsOp}(${mri}){${metricsQuery || ''}}`}</strong>,
```

Comment:
There's this helper that you might find useful:
https://github.com/getsentry/sentry/blob/d23ed4797b4a122b40cfc6df3370c80b354a1979/static/app/utils/metrics/index.tsx#L355